app/services/allocations/finder.rb

```diff
@@ -35,8 +37,24 @@ def apply_ordering(scope)
       end
     end
 
-    def split_params(name)
-      filter_params[name]&.split(',')
+    def apply_search(scope)
+      scope = apply_location_search(scope)
+      scope = apply_person_search(scope)
+      scope
+    end
+
+    def apply_location_search(scope)
+      return scope unless search_params.key?(:location)
+
+      scope.where(from_location_id: Location.search_by_title(search_params[:location])).or(
+        scope.where(to_location_id: Location.search_by_title(search_params[:location])),
+      )
+    end
+
+    def apply_person_search(scope)
+      return scope unless search_params.key?(:person)
+
+      scope.joins(moves: { profile: :person }).where('people.id' => Person.search_by_last_name(search_params[:person]))
```

Comment:
I think this might need to be an `includes` as the `joins` is ignoring the other scopes